platform/engineering/backend/review-instances.md

```diff
@@ -37,6 +37,58 @@ You will need your browser configured to access the vetsgov-internal domain via
 1. Specify the branch names for `api_branch` and `web_branch`. These branches will be deployed together with the review instance.
 1. When the process is completed, the URL for the review instance will be provided at the end of the output logs.
 
+### Access and Usage
+
+**NOTE:** This section is actively being worked on right now and these processes should get easier as time goes on. Some of these processes are extremely painful at the moment, but we are making the feature available in its rough form as these get worked out. The epic for this work can be found [here](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/4617), please reach out to #vfs-platform-support for feature requests or questions.
+
+These instances are running on an independent virtual machine that includes an installation of both vets-api and vets-website with an nginx proxy in front getting requests to the right place.
+
+Anyone who is cleared by their COR to work on VSP and has already has access to the internal tools should be able to access these instances via SSH. Once logged in, you can modify the code already checked out and manipulate the review instances as necessary.
```

Comment:
```suggestion
The instances are accessible via SSH behind the [SOCKS proxy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#configure-the-socks-proxy) and code can be freely modified in place. See [Internal Tools Access](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#internal-tools-access) to gain access.
```

I'm not sure we need to be calling things like COR and VSP - we've done that in a lot of other places already.

Reply:
Completely agree with you Bill